web-server/views/breadcrumb.jade

```diff
@@ -3,6 +3,8 @@ ol.breadcrumb.bg-light-grey
     a(href='/') home
 
   - breadcrumbs = path.split('/')
+  if breadcrumbs[breadcrumbs.length-1][0] === '?'
```

Comment:
what if we change https://github.com/GuildCrafts/curriculum/blob/master/web-server/helpers.js#L64 instead to something like this:

```js
response.path = request.url.split('?')[0]
```